_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -69004,7 +69004,6 @@
 /area/medical/virology)
 "cNt" = (
 /obj/machinery/button/crematorium{
-	id = "cremawheat";
```

Comment:
Add the ID to the crematorium instead, otherwise this can/will trigger -all- crematoriums without an ID assigned.